Sources/Apodini/Properties/Throws.swift

```diff
@@ -11,34 +11,65 @@ import Foundation
 @propertyWrapper
 public struct Throws {
     private let options: PropertyOptionSet<ErrorOptionNameSpace>
-    
+
     private let `type`: ErrorType
     
     private let reason: String?
     
     private let description: String?
+
+    private let information: InformationSet
     
-    internal init(type: ErrorType, reason: String? = nil, description: String? = nil, _ options: [ApodiniError.Option]) {
+    internal init(
+        type: ErrorType,
+        reason: String? = nil,
+        description: String? = nil,
+        information: [AnyInformation] = [],
+        _ options: [ApodiniError.Option]
+    ) {
         self.options = PropertyOptionSet(options)
         self.type = `type`
         self.reason = reason
         self.description = description
+        self.information = InformationSet(information)
     }
-    
+
     /// Create a new `ApodiniError` from its base components:
     /// - Parameter `type`: The associated `ErrorType`. If `other` is chosen, the `options` should be
     ///   used to provide additional guidance for the exporters.
     /// - Parameter `reason`: The **public** reason explaining what led to the this error.
     /// - Parameter `description`: The **internal** description of this error. This will only be exposed in `DEBUG` mode.
     /// - Parameter `options`: Possible exporter-specific options that provide guidance for how to handle this error.
-    public init(_ type: ErrorType, reason: String? = nil, description: String? = nil, _ options: ApodiniError.Option...) {
+    public init(
+        _ type: ErrorType,
+        reason: String? = nil,
+        description: String? = nil,
+        _ options: ApodiniError.Option...
+    ) {
         self.init(type: type, reason: reason, description: description, options)
     }
     
+    /// Create a new `ApodiniError` from its base components:
+    /// - Parameter `type`: The associated `ErrorType`. If `other` is chosen, the `options` should be
+    ///   used to provide additional guidance for the exporters.
+    /// - Parameter `reason`: The **public** reason explaining what led to the this error.
+    /// - Parameter `description`: The **internal** description of this error. This will only be exposed in `DEBUG` mode.
+    /// - Parameter `information`: Possible array of `Information` entries attached to the `Response`.
+    /// - Parameter `options`: Possible exporter-specific options that provide guidance for how to handle this error.
+    public init(
+        _ type: ErrorType,
+        reason: String? = nil,
+        description: String? = nil,
+        information: AnyInformation...,
+        options: ApodiniError.Option...
+    ) {
```

Comment:
I am fine with adding an extra convenience initializer where you can pass in information directly, but I would suggest to make InformationSet a PropertyOption, so we keep the type as slim as possible. We have so many strong generic patterns in Apodini but tend to not use them for implementing our internal features.

Reply:
Not sure about that one. I think it is important to distinguish the uses of Options and Information.
I looked at `Options` (in the case of ApodiniError) always as a way of supplying metadata for the exporters which gives guidance **how** to handle a given request, as it states in the documentation: https://github.com/Apodini/Apodini/blob/864a13505d2a5cdedca487a1e49dde2cf4cbd094/Sources/Apodini/Response/ApodiniError.swift#L53

In contrast `Information` actually delivers **content** for the generated response. They provide something instead of just giving guidance.

So I'm not sure about declaring that Informations are an Option just for the sake of removing a stored property(?).

Reply:
Good point. I guess the lines are a bit blurry around things like status codes, but taking that argument into account I am fine with leaving it as is.